option optimize_for = LITE_RUNTIME;

message LuxurySignGetRewardRequest
{
 required uint32 id = 1;
}
